<launch>
  
  <!-- LAUNCH INTERFACE FOR FT SENSORS -->
  <arg name="name" doc="Name of the sensor for unique identification, it depends on your URDF implementation. Mandatory."/>
  <arg name="type" doc="Type of sensor, e.g. nano17, it depends on your URDF implementation, check which models this package supports. Mandatory."/>

  <!-- LAUNCH IMPLEMENTATION -->
  <group ns="$(arg name)">
    <node  name="simulation_ft_sensor_hw" pkg="force_torque_sensor_controller" type="force_torque_sensor_interface" respawn="true" output="screen">
<!--       <param name="name" value="$(arg name)" />
      <param name="type" value="$(arg type)" /> -->
    </node>
  </group>
   
</launch>